configs/backstage.json

```diff
@@ -1,12 +1,23 @@
 {
   "index_name": "backstage",
   "start_urls": [
-    "https://backstage.io/docs/"
-  ],
-  "sitemap_urls": [
-    "https://backstage.io/sitemap.xml"
```

Comment:
The sitemap should still be present if available, it helps the scraper to find URLs but follows the `start_urls` structure

Reply:
Reading the docs it wasn't very clear, https://docsearch.algolia.com/docs/config-file/#sitemap_urls-optional
> DocSearch will try to read URLs from your sitemap(s) instead of following every link of your starts_urls.
I'll add it back

Reply:
Thanks for the feedback, a new documentation is on the way https://github.com/algolia/docsearch/pull/1048

You can submit an alternative wording if you want!